### DISCUSSION TITLE
possible to add a description to a collection type?

### DISCUSSION THREAD

**Discussion by tomhallio** [strapi/strapi#6982] [Legacy Topics]
Simple question, is it possible to add a description to a collection type?

<img width="227" alt="Screenshot 2020-07-12 at 11 48 06" src="https://user-images.githubusercontent.com/38839533/87244480-a93ed700-c435-11ea-873e-2819c4009e7d.png">

There is a default value; "There is no description", but I would like to change it.

**Comment by MattieBelt** [comment]
`api/task/models/task.settings.json`
https://strapi.io/documentation/v3.x/concepts/models.html#model-information
